# Define the goal
# Time to try to detect the fire (in seconds. If 0 detect until success)
std_msgs/Duration detect_time
---
# Define the result
bool fire_found
geometry_msgs/Point fire_position
std_msgs/Float64 errorX
std_msgs/Float64 errorY
std_msgs/Float64 errorZ

 
---
# Define a feedback message
#Current time since the begining (in seconds)
std_msgs/Duration current_time


